build.sbt

```diff
@@ -149,7 +124,10 @@ lazy val s3sdkExamples = (project in file("examples-s3-sdk-extension"))
     (Compile / compile) := ((Compile / compile) dependsOn printSparkScalaVersion).value, // printSparkScalaVersion is run with compile
     mergeStrategy,
   )
-  .settings(jacocoS3sdkExamplesReportSettings: _*)
+  .settings(
+    jacocoReportSettings := commonJacocoReportSettings.withTitle("atum:s3sdkExamples Jacoco Report"),
+    jacocoExcludes := commonJacocoExcludes
+  )
```

Comment:
Examples should not have a Jacoco report. We are even skipping tests in them. They are not a deliverable package but rather a showcase.